frontend/src/app/list/components/list-details/list-item-dialog/list-item-dialog.component.html

```
@@ -1,6 +1,4 @@
-<h2 mat-dialog-title>
-  {{ data.listItem ? "Editar" : "Adicionar" }} Item na Lista
-</h2>
+<h2 mat-dialog-title>{{ data.listItem ? "Edit Item" : "Add Item" }} to List</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)